homeassistant/components/youless/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for youless integration."""
+import logging
+from urllib.error import HTTPError, URLError
+
+import voluptuous as vol
+from youless_api import YoulessAPI
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_DEVICE, CONF_HOST, CONF_NAME
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({vol.Required(CONF_NAME): str, vol.Required(CONF_HOST): str})
```

Comment:
What is name? Do we need to ask for that?

Reply:
The name is used for generating unique ids in the various sensors. The name is included in the id to enable the adding of multiple Youless devices with ids that make sense.

Reply:
I have changed the unique device ID to use the MAC address if available or a UUID from config_utils when unavailable.

The only reason to keep name is for displaying in the UI to identify which activated YouLess integration you are displaying. This is in the integrations screen, the name of the integration. I do not know of any other elegant way to identify the integration otherwise, since the devices do not expose any name. Especially for users that activate multiple instances of Youless because they have multiple devices.